api/build.gradle

```diff
@@ -12,6 +10,7 @@ buildscript {
     }
     dependencies {
         classpath "gradle.plugin.org.inferred:gradle-processors:2.2.0"
+        classpath 'com.github.dcendents:android-maven-plugin:1.2'
```

Comment:
I'm not an expert on Android by any means, but googling this library comes up with https://github.com/dcendents/android-maven-plugin which says you should be using https://github.com/simpligility/android-maven-plugin instead but we're using Gradle so I followed that path and it says the android-library plugin should be used instead? https://developer.android.com/studio/publish-library

Reply:
Tons of improvements can happen, this line is result of this (same PR):

![image](https://user-images.githubusercontent.com/6842786/165728712-f87f3d5e-ed69-4645-b712-8df0647b81f7.png)

so I only updated the way it's being used without affecting any other dependencies

Reply:
Ok, we'll come back to that when we have more time to upgrade the project